moebot_bot/util/db/channelRotation.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"log"
+	"strings"
+)
+
+type ChannelRotation struct {
+	ChannelUIDList    []string
+	CurrentChannelUID string
+	ScheduledOperation
+}
+
+const (
+	channelRotationTable = `CREATE TABLE IF NOT EXISTS channel_rotation(
+		operation_id INTEGER NOT NULL PRIMARY KEY REFERENCES scheduled_operation(id) ON DELETE CASCADE,
+		current_channel_uid VARCHAR(20) NOT NULL,
+		channel_uids VARCHAR NOT NULL
```

Comment:
If we don't have a limit let's go with `TEXT` instead.
I'd also prefer a limit (`TEXT` with a `CHECK` on it's length) so that there's at least something preventing people from creating tons and tons of these rotating channels, even if that limit is super high.

Reply:
TEXT and VARCHAR are the same thing internally, but we can go with TEXT if you prefer that naming for consistency reasons, no problem.
Instead, if you want a limit, why not going for VARCHAR(n)?